README.md

```diff
@@ -85,6 +85,12 @@ The resulting recorded snapshot files are currently expected to be checked into
 used to inform `upgrade-quick` and `upgrade-preview-only` tests. When updating the baseline version,
 snapshots need to be recorded anew on the new version.
 
+### Fixing failing tests
+If a test fails after changing it then the snapshot files likely need to be re-recorded.  
```

Comment:
Oof, I think this statement "If a test fails after changing it then the snapshot files likely need to be re-recorded." is exactly false, but perhaps we're mis-communicating one more time. Can you elaborate? When does a failing test lead to the need to re-record baselines..